tests/testthat/test-estimate_KM.R

```diff
@@ -22,6 +22,7 @@
 #' T3.3 When no strata are specified, an artificial strata is created 'Overall'
 #' T3.4 When only 1 stratum is specified, the stratum names are added to the `names` attribute'
```

Comment:
We already had strata labels in the names attribute of survfit_object[["strata"]]. Can we not leverage that logic instead of adding another list to the survfit_object? Sorry to notice it this late.

For me its to include another element, but it seems overkill.

Reply:
that object was not modified in this PR, and those are the strata levels